game/scripts/npc/abilities/storm_spirit_ball_lightning.txt

```diff
@@ -19,15 +19,15 @@
 
     // Casting
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCastPoint"                                    "0.4" // OAA
+    "AbilityCastPoint"                                    "0.45 0.42 0.4 0.35 0.3" //OAA
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "30"
+    "AbilityManaCost"                                     "10 20 30 40 50" //OAA
 
     // Damage
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityDamage"                                       "8 12 16 20 24"
+    "AbilityDamage"                                       "8 12 16 32 48"
```

Comment:
if the point is to make the storm engage more, why make the move where he's invulnerable do 2x damage?

Reply:
Buff to dmg is to encourage people more to upgrade the ability, because someone told me that if mana cost increases people will not want to upgrade it. Can be easily reverted.